keras/src/utils/jax_layer.py

```diff
@@ -196,6 +214,9 @@ def my_haiku_module_fn(inputs, training):
         init_fn: the function to call to initialize the model. See description
             above for the list of arguments it takes and the outputs it returns.
             If `None`, then `params` and/or `state` must be provided.
+        compute_output_shape_fn: Function that takes the input shape
+            (a tuple or nested structure of tuples) and returns the output
+            shape (a tuple or nested structure of tuples).
```

Comment:
Why was this needed? Also, it looks like it's for TensorFlow only?

If `compute_output_shape` is not implemented, Keras should trace the layer and figure it out automatically, which is a better user experience.